doc/source/whatsnew/v1.0.0.rst

```diff
@@ -356,6 +356,23 @@ When :class:`Categorical` contains ``np.nan``,
 
    pd.Categorical([1, 2, np.nan], ordered=True).min()
 
+
+Default dtype of empty :class:`pandas.Series`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
the underline needs to be the same length as the text; this is supposed to cause a doc-warning, weird that it is not @jorisvandenbossche

Reply:
Sphinx only warns when it is shorter ;) 
(that said, in pandas we try to keep it the exact same length indeed)